src/main/java/org/jboss/ejb/client/DeploymentNodeSelector.java

```diff
@@ -36,8 +39,10 @@
  * yields a URI or cluster, this mechanism is not used.
  *
  * @author Jaikiran Pai
+ * @author <a href="mailto:wfink@redhat.com">Wolf Dieter Fink</a>
  */
 public interface DeploymentNodeSelector {
+	final static Logger log = Logger.getLogger(DeploymentNodeSelector.class);
```

Comment:
Can you replace this one with Logs.Main as well?

Reply:
@chengfang fixed and updated PR